### DISCUSSION TITLE
red_heart gone

### DISCUSSION THREAD

**Discussion by Timoyoungster** [d99kris/nchat#279] [Q&A]
Hello,
I am missing the :red_heart: emoji when trying to send it from nchat, but if I send it from my phone it gets shown without problems.
Have I misconfigured something or is that a known issue?

**Comment by d99kris** [comment]
Hello @Timoyoungster - this is currently an intentional limitation. `:red_heart:` (and several other emojis) do not work well in common terminals, so they have been excluded from the nchat emoji selector. You can see the issue by copy/pasting a `:red_heart:` into nchat when writing a message (or into other terminal apps as well, like nano), then basically the position of any text following the emoji will be slightly off (at least in macOS Terminal and Ubuntu gnome-terminal). _Possibly_, we could consider adding a config option in nchat to make all emojiis available, in case there is some Terminal which renders it properly..

In place of `:red_heart:` you may consider using `:heart:` - I believe it renders the same in official Telegram and WhatsApp apps - although it looks different in the terminal.

**Reply by Timoyoungster** [reply]
Oh, ok thank you for the information.
Is there an issue i can subscribe to, to get updates on that matter?

**Reply by d99kris** [reply]
There's no other issue reported, I can post in this thread when/if there is any update. _However_, I don't think the basic problem (of text after `:red_heart:` rendering at wrong position) should be worked around in nchat. Ideally it should be fixed at its root cause.

Having that said, please let me know if you think it would be useful to have an option to enable all emojis (incl. `:red_heart:`) in the nchat emoji selector? This could be done, but it would not fix the test position issue mentioned.

**Reply by Timoyoungster** [reply]
I think an option would be great.
Since it still displays the :red_heart:'s I receive, I don't mind / have the issue all the time anyways.

**Reply by d99kris** [reply]
> I think an option would be great. Since it still displays the :red_heart:'s I receive, I don't mind / have the issue all the time anyways.

Ok! In f55d9a4 an option `emoji_list_all` in `app.conf` has been added, if setting it to `1` the full list of emojis will be listed in the emoji selector dialog (incl. `:red_heart:`).

**Reply by Timoyoungster** [reply]
Thank you!
